gobblin-core/build.gradle

```diff
@@ -51,7 +51,7 @@ dependencies {
 
   if (project.hasProperty('useHadoop2')) {
     compile externalDependency.avroMapredH2
-    testRuntime externalDependency.hadoopAws
+    compile externalDependency.hadoopAws
```

Comment:
I changed it to a compile dependency because it is needed to use the s3a filesystem.  Support for the s3a filesystem was the point of this PR.